packages/elements/src/chart/elements/chart.ts

```diff
@@ -333,8 +327,29 @@ export class Chart extends BasicElement {
     // set global config again in case the CSS font isn't loaded when updating the chart
     this.setGlobalConfig();
     this.decorateColors(chart);
+    this.decorateGridColors(chart);
   };
 
+  /**
+   * Set grid color in beforeUpdate event from Chartjs
+   * @param chart Chart.js instance
+   * @returns {void}
+   */
+  private decorateGridColors(chart: ChartJS): void {
+    for (const scale in chart.scales) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn#iterating_over_the_properties_of_an_object
```suggestion
    for (const scale of Object.keys(chart.scales)) {
```